src/content/pages/backpacking-checklist.md

```diff
@@ -1,5 +1,110 @@
 ---
-title: "Backpacking checklist"
+title: "Backpacking Checklist"
 description: "this is meta description"
 draft: false
----
\ No newline at end of file
+--- ### Core Backpacking Needs
```

Comment:
The article text should normally start below the metadata section above. Like this:

```
---
title: "Backpacking Checklist"
description: "this is meta description"
draft: false
---

### Core Backpacking Needs
```